data/data/bootstrap/baremetal/files/etc/metal3.env.template

```diff
@@ -0,0 +1,11 @@
+AUTH_DIR=/opt/metal3/auth
+HTTP_PORT=6180
+# This DHCP range is used by dnsmasq to serve DHCP to the cluster. If empty
+# dnsmasq will only serve TFTP, and DHCP will be disabled.
+DHCP_RANGE="{{.PlatformData.BareMetal.ProvisioningDHCPRange}}"
+DHCP_ALLOW_MACS="{{.PlatformData.BareMetal.ProvisioningDHCPAllowList}}"
+# Used by ironic to allow ssh to running IPA instances
+IRONIC_RAMDISK_SSH_KEY="{{.SSHKey}}"
+IPTABLES={{if .PlatformData.BareMetal.ProvisioningIPv6}}ip6tables{{else}}iptables{{end}}
```

Comment:
Is this file used only for BMO? If yes, I suspect only AUTH_DIR and IRONIC_ENDPOINT are useful.

Reply:
Done